### ISSUE TITLE
Make article GitHub source repository more prominent

### ISSUE THREAD

**Issue by j3mdamas** [livecomsjournal/livecomsjournal.github.io#196]
Hi all,

First of all, I love the concept of perpetual articles that evolve along time. Congratulations for the initiative.

I have a small pet peeve: from the [journal page](https://www.livecomsjournal.org), on each article, it is very hard to imediately reach the source repository. I see it in the PDF, next to the Abstract, but I think it should rather have a more prominent place, not only in the PDF, but also in the HTML page of the article.

Cheers,
João

**Comment by davidlmobley** [comment]
On the HTML page is a great suggestion.

Are you suggesting we should also make it more prominent on the PDF? We'd decided the left margin made it the most prominent of the options we considered, but are open to suggestions.

cc @mrshirts

**Comment by j3mdamas** [comment]
Yes, I was also suggesting a more prominent place in the PDF, but I agree with you that it's not easy to find a better place. My initial impression was: "OK, if I want to contribute to this paper, where do I go? It's on GitHub, it must be somewhere." And it took me a moment before realizing that italic text next to abstract was had what I was looking for. Perhaps the HTML would be enough.
